### DISCUSSION TITLE
thanhle.blog/ban-muon-doc-ve-chu-de-nao

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#85] [Ideas]
# thanhle.blog/ban-muon-doc-ve-chu-de-nao

Không biết viết gì cả

https://thanhle.blog/blog/ban-muon-doc-ve-chu-de-nao

**Comment by khiem20tc** [comment]
Em xin vote cho 2 bài "trải nghiệm một mình" và "dev level quốc tế" nhé anh. Chấm hónggg : )))

**Reply by thanhlmm** [reply]
<img width="823" alt="CleanShot 2023-10-18 at 11 52 38@2x" src="https://github.com/thanhlmm/blog/assets/9281080/fa75d876-dc70-4d11-adde-1f83e5697041">
Gửi e :)))

**Reply by khiem20tc** [reply]
vcl đã dev xịn lại còn làm thơ hay thế anh =)))

**Reply by thanhlmm** [reply]
a mở ra cũng bất ngờ ko biết hôm này mà làm thơ ác z :))

**Reply by ttdung997** [reply]
em cũng vote dev level quốc tế

**Comment by tan12399** [comment]
Em muốn xem "Ký sự Freelance ở Upwork", em đang gặp vấn đề về việc kiếm job ngoài cty nhưng k biết làm thế nào, mong anh chia sẻ thêm. Hoặc là anh nói về vấn đề liên tục các bên gỡ bỏ Typescript đi ạ. Cám ơn anh ✌️

**Reply by thanhlmm** [reply]
A có bài này rồi mà nhỉ https://thanhle.blog/blog/ki-su-freelancer-o-upwork

**Reply by tan12399** [reply]
Vâng, nhưng là vì có phần 2 nên em muốn biết thêm 😁

**Comment by hai123vn** [comment]
em chọn "Đừng làm việc chăm chỉ, hãy làm việc hiệu quả" nha a choai ơi

**Reply by hai123vn** [reply]
dạo này bận quá hả anh ơi =))

**Reply by thanhlmm** [reply]
dạo này vẫn giống dạo xưa, vẫn chăm gáy bẩn vẫn chăm bùng bài 🤷‍♂️

**Reply by hai123vn** [reply]
lẹ lẹ ra bài nha anh vẫn hóng từng ngày luôn á =))

**Comment by nguyenhoaikhang37** [comment]
Biết blogs của anh đang trong giai đoạn drop nhưng vẫn có thói quen 1 tuần vào 1 lần, vì đọc blog anh từ năm trc tới h. Thấy anh ra bài này cũng vui vì cũng nhớ ae fan 20 năm như em kk. Sắp tới nếu tg ra 1 blog kĩ thuật khá mất time, em nghĩ chuyển qua làm thơ chém gió th anh (thấy trình độ làm thơ anh khác xưa gòi :)) 
Như tên title hnay, em thấy kết các bài như
- Một vài suy nghĩ khi bắt tay code 1 features
- Anh hùng bàn phím để trở nên productive hơn
- Thái độ khi đi làm là gì
à, còn 1 vài bài em còn băn khoăn
- Làm sao giao tiếp vs foreigner hiệu quả (em nhớ có bài anh bận nên dời deadline lại nhưng KH vẫn respect)

**Reply by thanhlmm** [reply]
Có bài mới rồi kìa em :D

**Comment by PhongLi** [comment]
Nay ngồi tìm đọc lại mấy bài blockchain a viết thì thấy có bài bay-gio-hoc-blockchain-lieu-co-muon 404 mất tiêu ._. Sao a xóa vậy ạ?

**Reply by thanhlmm** [reply]
Hic anh cũng ko biết sao bị 404 nữa. Em đọc lại ở đây nhé https://thanh-le.notion.site/B-y-gi-h-c-blockchain-li-u-c-mu-n-cbd7eef25e5f494c843022852f82d175?pvs=4

Có time để anh debug thử

**Reply by PhongLi** [reply]
Thanks a. Hóng thêm nhiều bài blockchain từ anh ạ ( về kinh nghiệm làm việc hay quá trình tự học...).🤩

**Comment by nguyenquangcr** [comment]
Em hóng bài Frontend architecture và một vài comment trong Nextjs13.
Cá nhân a thấy giờ làm frontend còn hot và còn phát triển trong vài năm tới không ạ .

**Reply by thanhlmm** [reply]
Nextjs 13 hình như anh đã có bài review rồi thì phải

Cá nhân mình nghĩ frontend trong vòng 3 năm tới sẽ ko còn nhiều phát triển kiểu đột phá nữa, chủ yếu là xoay quanh những thứ đang có và làm cho nó ổn định hơn thôi
